dts/bindings/mmc/st,stm32-sdmmc.yaml

```diff
@@ -0,0 +1,22 @@
+description: stm32 sdmmc disk access
+
+compatible: "st,stm32-sdmmc"
+
+include: mmc.yaml
+
+properties:
+    clocks:
+        required: true
+    label:
+        type: string
```

Comment:
shouldn't need `type: string` here.

Reply:
I assume I should make a new pull request for that? Or can I still update this one after it's merged?

Reply:
Fixed in #25132